tasty-query/jvm/src/test/scala/tastyquery/testutil/jdk/JavaTestPlatform.scala

```diff
@@ -13,18 +13,21 @@ object JavaTestPlatform {
   private val TestClassPathEnvVar = "TASTY_TEST_CLASSPATH"
   private val ResourceCodeEnvVar = "TASTY_TEST_SOURCES"
 
-  private lazy val classpath: Classpath =
-    val stringEntries = System.getenv(TestClassPathEnvVar).nn.split(';').toList
+  private lazy val classpathEntries: List[String] =
+    System.getenv(TestClassPathEnvVar).nn.split(';').toList
+
+  private lazy val classpathPaths: List[Path] =
+    for stringEntry <- classpathEntries yield stringEntry match
+      case s"jrt:/modules/$module/" =>
+        FileSystems.getFileSystem(java.net.URI.create("jrt:/")).nn.getPath("modules", module).nn
+      case _ =>
+        Paths.get(stringEntry).nn
 
-    val entries: List[Path] =
-      for stringEntry <- stringEntries yield stringEntry match
-        case s"jrt:/modules/$module/" =>
-          FileSystems.getFileSystem(java.net.URI.create("jrt:/")).nn.getPath("modules", module).nn
-        case _ =>
-          Paths.get(stringEntry).nn
+  private lazy val classpath: Classpath =
+    ClasspathLoaders.read(classpathPaths)
 
-    ClasspathLoaders.read(entries)
-  end classpath
+  lazy val scala3ClasspathIndex: Int =
+    classpathEntries.indexWhere(_.contains("scala3-library_3").nn)
```

Comment:
This could encapsulate better the `Int`-based lookup:
```suggestion
  lazy val scala3ClasspathEntry: Classpath.Entry =
    classpath.entries(classpathEntries.indexWhere(_.contains("scala3-library_3").nn))
```
Same in `TestPlatform` and on the `js/` side.

Reply:
I chose not to do this as it would have to be `Future[Classpath.Entry]` - and test suite can do unsafe stuff